app/src/main/assets/ime/keyboard/org.florisboard.localization/extension.json

```
@@ -644,6 +645,15 @@
       "preferred": {
         "characters": "org.florisboard.layouts:urdu_phonetic"
       }
-    }
+    },
+    {
+      "languageTag": "ast",
+      "composer": "org.florisboard.composers:appender",
+      "currencySet": "org.florisboard.currencysets:euro",
+      "popupMapping": "org.florisboard.localization:ast",
+      "preferred": {
+        "characters": "org.florisboard.layouts:qwerty"
+      }
+    },
```

The last comma after the curly brace is invalid JSON and causes a parse error, removing it fixes the problem.
 
```suggestion
    }
```